plugin/src/main/java/org/gradle/testretry/internal/executer/RetryTestExecuter.java

```diff
@@ -120,7 +120,7 @@ public void execute(JvmTestExecutionSpec spec, TestResultProcessor testResultPro
     }
 
     public void failWithNonRetriedTestsIfAny() {
-        if (extension.getSimulateNotRetryableTest() || (lastResult != null && !lastResult.nonRetriedTests.isEmpty())) {
+        if (extension.getSimulateNotRetryableTest() || lastResult != null && !lastResult.nonRetriedTests.isEmpty()) {
```

Comment:
Was there an IDEA warning for these? I think we should instead move the `&&` expression to keep this readable.

Reply:
The checkstyle task fails without the change: https://e.grdev.net/s/7wkocu7ovlrhc/failure#1.

Reply:
I guess that's because the default `toolVersion` changed recently: https://github.com/gradle/gradle/commit/b4d55d51d7cc9ae971ef47b0ff66b94c7a365ea7

Reply:
Done in 9d3cbe71ea7ed1e4aff5b685d2b96caf3a97cea5.